### DISCUSSION TITLE
does learn.calibrate_model() work for regression model?

### DISCUSSION THREAD

**Discussion by zmce2018** [timeseriesAI/tsai#530] [Q&A] [answered]
Hi all, 

I am very new to deep learning and have recently found this API. I am wondering if  learn.calibrate_model() work for regression model?  

Kind regards.

**Comment by oguiza** [comment, marked as answer]
Hi @zmce2018,
No, it wouldn't work. It only works when the model outputs probabilities. 
I don't think that model calibration is applicable to regression problems. In regression, you are only interested in getting a prediction that is as close as possible to the target.
